pom.xml

```diff
@@ -503,37 +508,6 @@
                     <version>1.3</version>
                 </plugin>
                 <plugin>
-                    <groupId>org.bsc.maven</groupId>
```

Comment:
For what's it worth, this works nicely in Eclipse :)

> Since other principles already weakened (see Eclipse pulgin settings)

That's two different things, though. The added plug-in config doesn't do any harm (apart from violating your sense of aesthetics ;) whereas this dependency really is wrong in the sense that it allows to (accidentally) import stuff from the processor JAR.

But in the end I don't care much about it, tests would show up this sort of issues anyways. So if you think that's the lesser evil, let's do it in order to move forward.